src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -136,12 +137,71 @@ public ViewMode CurrentMode
         }
 
         /// <summary>
-        ///     SearchResults property
+        ///     Items that were found during search.
         /// </summary>
-        /// <value>
-        ///     This property is observed by SearchView to see the search results
-        /// </value>
-        public ObservableCollection<NodeSearchElementViewModel> SearchResults { get; private set; }
+        private List<NodeSearchElementViewModel> searchResults;
+
+        private IEnumerable<NodeSearchElementViewModel> filteredResults;
+        /// <summary>
+        /// Filtered search results.
+        /// </summary>
+        public IEnumerable<NodeSearchElementViewModel> FilteredResults
```

Comment:
There is no need to make FilteredResults `ObservableCollection`, because we notify view by calling `RaisePropertyChanged`.